src/query/service/src/interpreters/interpreter_merge_into.rs

```diff
@@ -15,6 +15,7 @@
 use std::collections::HashMap;
 use std::sync::Arc;
 use std::time::Instant;
+use std::usize::MAX;
```

Comment:
better to use determined integer type: u32 or u64

Reply:
ok